tests/json-fixtures/test_transactions.py

```diff
@@ -80,12 +78,21 @@ def test_transaction_fixtures(fixture_name, fixture):
 
         sender = to_canonical_address(fixture['sender'])
 
-        assert transaction.sender == sender
+        try:
+            assert transaction.sender == sender
+        except BadSignature:
+            assert not (27 <= transaction.v <= 34)
     else:
         # check RLP correctness
         try:
             transaction = rlp.decode(fixture['rlp'], sedes=TransactionClass)
-        except rlp.exceptions.ObjectDeserializationError:
+        except KeyError:
+            assert fixture['rlpHex']
+            return
+        except TypeError as err:
+            assert err.args == ("'bytes' object cannot be interpreted as an integer",)
```

Comment:
In fact, can you toss a comment into each of these except blocks with a brief explanation of what they are catching.